{"name": "test_add_new_student", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'H:\\\\Git_projekty\\\\Api using python\\\\add_student.json'", "trace": "def test_add_new_student():\n        global id\n        url = 'https://thetestingworldapi.com/api/studentsDetails'\n>       f = open(os.path.abspath('add_student.json'), 'r')\nE       FileNotFoundError: [Errno 2] No such file or directory: 'H:\\\\Git_projekty\\\\Api using python\\\\add_student.json'\n\nTestSMS\\test_requests_chaining.py:9: FileNotFoundError"}, "start": 1669149472059, "stop": 1669149472060, "uuid": "305db028-ecc9-40d8-a453-557b425e70ab", "historyId": "5eb780d38d258f7e73d75ca5720933f9", "testCaseId": "7dc1def3dd7a189ad790375395ede9d6", "fullName": "TestSMS.test_requests_chaining#test_add_new_student", "labels": [{"name": "parentSuite", "value": "TestSMS"}, {"name": "suite", "value": "test_requests_chaining"}, {"name": "host", "value": "DESKTOP-3351BCE"}, {"name": "thread", "value": "1952-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSMS.test_requests_chaining"}]}